src/app/core/services/api/fake-backend/catalog-fake-service.ts

```diff
@@ -0,0 +1,38 @@
+import {UiDataOffer} from '@sovity.de/edc-client';
+import {TestAssets} from './data/test-assets';
+import {TestPolicies} from './data/test-policies';
+
+export let uiDataOffers: UiDataOffer[] = [
```

Comment:
when not explicitly differentiating between a "uiDataOffer" and a "dataOffer", I think we should leave the prefixes away, it doesn't seem to do us any good, also, for example with the "uiAsset" parameters, in the end, having it just called "asset" would make things better... imho